### ISSUE TITLE
Discussion about License

### ISSUE THREAD

**Issue by jy1909** [UBC-MDS/text_processing_util_mds24#70]

**Comment by MoNorouzi23** [comment]
I think MIT can be the best option for out package because the MIT License is highly permissive, allowing users to do almost anything they want with the code. Also, It imposes minimal restrictions on how the software can be used, modified, and distributed as well as the license is short, simple, and easy to understand. It is compatible with other licenses, allowing us to combine MIT-licensed code with code under different licenses.

**Comment by jy1909** [comment]
I think we should keep the MIT License as well. We are not working in a specific community, and I do not really see why we need to ask others keep their code open source for commercial purposes. We just need to keep it simple and we do not need to impose a lot of restrictions.

**Comment by allan8392** [comment]
I agree to using MIT license. We do not want to be responsible for any error in our software. Furthermore I am ok with anyone reuse our software for any purpose.

**Comment by nassimgha** [comment]
I also agree to using the MIT License,  as it is a widely-used, permissive open-source license that is simple to understand.
